src/main/webapp/style.css

```diff
@@ -44,6 +44,43 @@
   display: none;
 }
 
+#routeMap {
+  border: thin solid black;
+  height: 500px;
+  width: 700px;
+  display: inline-block;
+}
+
+#rightPanel {
+  font-family: 'Roboto','sans-serif';
+  line-height: 30px;
+  padding-left: 10px;
+  height: 500px;
+  width: 300px;
+  display: inline-block;
+  margin-left: auto;
+  margin-right: auto;
+  overflow: auto;
+
+}
+
+#rightPanel select, #rightPanel input {
+  font-size: 15px;
+}
+
+#rightPanel select {
+  width: 100%;
+}
+
+#rightPanel i {
+  font-size: 12px;
+}
+
+#content {
+  overflow: hidden;
+  text-align: center;
+}
+
```

Comment:
I really like the styling! Good job

Reply:
Just as an fyi I like to use a class="name" rather than id="name" because it makes it easier to clearer (in my opinion) to style a div within a div (so .rightPanel .topPart a). Just a personal preference wanted to thought I would let you know about the option if you didn't already.

Reply:
That's a good point. I'll change this in a future CL.